libs/expo/betterangels/src/lib/hooks/interactions/useGetHmisClientInteractionsWithLocation.ts

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@apollo/client/react';
+import { HmisNoteOrdering, Ordering } from '../../apollo';
+import {
+  HmisNotesDocument,
+  HmisNotesQuery,
+} from '../../screens/ClientHMIS/tabs/ClientInteractionsHmisView/__generated__/ClientInteractionsHmisView.generated';
```

Comment:
not blocking, but I think it makes more sense to call these notes rather than interactions